libs/core/executors/include/hpx/executors/execution_policy.hpp

```diff
@@ -1622,5 +1622,19 @@ namespace hpx { namespace detail {
       : std::true_type
     {
     };
+
+    template <typename ExPolicy>
+    constexpr decltype(auto) async_to_sync_policy(ExPolicy&& policy)
+    {
+        if constexpr (hpx::is_async_execution_policy_v<ExPolicy>)
+        {
+            return hpx::execution::par.on(policy.executor())
+                .with(policy.parameters());
```

Comment:
This will always use `par` as the non-task policy. Is that acceptable?

Reply:
Ok yeah I should also consider if it's called somewhere else using a sequential policy. Btw are there other tasks policies besides seq(task) and par(task)? Also is this location okay or should I place this function somewhere else?

Reply:
Another option would be to add a `to_non_task()` member (or a `operator()(non_task)`) to all execution policies (similar to the member function that returns a corresponding task policy). Whatever is easier for you.